apps/src/turtle/turtle.js

```diff
@@ -1175,6 +1180,14 @@ Artist.prototype.step = function (command, values, options) {
       this.ctxScratch.restore();
 
       break;
+    case 'setArtist':
+      if (this.skin.id !== values[0]) {
+        this.skin = ArtistSkins.load(this.studioApp_.assetUrl, values[0]);
+        this.loadTurtle();
```

Comment:
It looks like [this calls `Artist.prototype.display` on `this.avatarImage.onload`](https://github.com/code-dot-org/code-dot-org/blob/be77b537e99650366a6dbd50c6bf531345508702/apps/src/turtle/turtle.js#L537).  Will we end up drawing pre-draw, etc. twice?